test/src/unit-capi-object_mgmt.cc

```diff
@@ -51,21 +51,21 @@ struct ObjectMgmtFx {
   const std::string S3_PREFIX = "s3://";
   const std::string S3_BUCKET = S3_PREFIX + random_bucket_name("tiledb") + "/";
   const std::string S3_TEMP_DIR = S3_BUCKET + "tiledb_test/";
-#ifdef _WIN32
+#if defined(_WIN32)
   const std::string FILE_URI_PREFIX = "";
   const std::string FILE_TEMP_DIR =
       tiledb::sm::Win::current_dir() + "\\tiledb_test\\";
   const std::string FILE_FULL_TEMP_DIR = FILE_TEMP_DIR;
-  const std::string GROUP = "group\\";
-  const std::string ARRAY = "array\\";
+//  const std::string GROUP = "group\\";
```

Comment:
Can you take out these commented lines if they're no longer needed?

Reply:
Oops, thanks. Dropped that commit when fixing formatting on my mac. Done.